tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -20,6 +20,7 @@ ENV LC_ALL=C.UTF-8
 ENV KMP_AFFINITY=granularity=fine,compact,1,0
 ENV KMP_BLOCKTIME=1
 ENV KMP_SETTINGS=0
+ENV MANUAL_BUILD=0
```

Comment:
Remove unused variable `MANUAL_BUILD `

Reply:
This variable is used internally during SMP build time